mucommander-core/src/main/java/com/mucommander/ui/dialog/debug/DebugConsoleDialog.java

```diff
@@ -328,29 +325,22 @@ private Color getColorForLevel(LogLevel level) {
             if (level == null) {
                 return Color.BLACK;
             }
-            Color color;
             switch (level) {
                 case SEVERE:
-                    color = Color.RED;
-                    break;
+                    return Color.RED;
                 case WARNING:
                     // Dark orange
-                    color = new Color(255, 100, 0);
-                    break;
+                    return new Color(255, 100, 0);
                 case CONFIG:
-                    color = Color.BLUE;
-                    break;
+                    return Color.BLUE;
                 case INFO:
-                    color = Color.BLACK;
-                    break;
+                    return Color.BLACK;
                 case FINE:
-                    color = Color.DARK_GRAY;
-                    break;
+                    return Color.DARK_GRAY;
                 default:
                     // Between Color.GRAY and Color.DARK_GRAY
-                    color = new Color(110, 110, 110);
-            };
-            return color;
```

Comment:
matter of taste, personally I like single point of return (unless it is return on the beginning of method - args check etc). Just a comment.

Reply:
yeah, that's subjective. I tend to prefer returning from each case when the switch-case is the very last element in a method